### PR TITLE
manual test for changing shipping option

### LINKED ISSUE #7245: Manually test that PaymentRequest attributes change
Need to check manually that each of these change or fire:

- [x] shippingAddress attribute
- [x] shippingType attribute
- [x] onshippingaddresschange attribute
- [x] shippingOption attribute
- [x] onshippingoptionchange attribute

### PR DESCRIPTION
**Pull request by marcoscaceres**
Closes #7245

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PASSED

Started: 2017-09-11 04:55:26
Finished: 2017-09-11 05:02:20


<h2>Failing Jobs</h2>
<ul>

<li>chrome:unstable</li>

<li>safari:10.0</li>

<li>MicrosoftEdge:14.14393</li>

</ul>




View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/18378)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/274040410)

File: payment-request/change-shipping-option-manual.https.html [hunk 1]
```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<!-- Copyright © 2017 Chromium authors and World Wide Web Consortium, (Massachusetts Institute of Technology, ERCIM, Keio University, Beihang). -->
+<meta charset="utf-8">
+<title>Test for PaymentRequest shippingOption attribute</title>
+<link rel="help" href="https://w3c.github.io/payment-request/#shippingoption-attribute">
+<link rel="help" href="https://w3c.github.io/payment-request/#onshippingaddresschange-attribute">
```
Comment (edenchuang) [inline payment-request/change-shipping-option-manual.https.html]: s/onshippingaddresschange-attribute/onshippingoptionchange-attribute

File: payment-request/change-shipping-option-manual.https.html [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+function testShippingOptionChanged() {
+  promise_test(async t => {
+    const detailsWithShippingOptions = Object.assign({}, validDetails, {
+      shippingOptions: [validShippingOption1, validShippingOption2],
+    });
+    const request = new PaymentRequest(
+      validMethods,
+      detailsWithShippingOptions,
+      requestShipping
+    );
```
Comment (edenchuang) [inline payment-request/change-shipping-option-manual.https.html]: Please add assertion for shippingOption initialization checking.
assert_equals(request.shippingOption, "fail", "request.shippingOption should be initialized with 'fail'")
Maybe using "default" is better than "fail".

**Review by edenchuang — APPROVED** [review]
couple of nits.

File: payment-request/change-shipping-option-manual.https.html (hunk 2 above)
Comment (marcoscaceres) [reply payment-request/change-shipping-option-manual.https.html]: Added this just below

**Pull request merged**